src/OpenTracing/ISpan.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace OpenTracing
+{
+    /// <summary>
+    /// Span represents an active, un-finished span in the OpenTracing system.
+    /// Spans are created by the <see cref="ITracer"/> interface.
+    /// </summary>
+    public interface ISpan : IDisposable
+    {
+        // TODO IDisposable behavior?
```

Comment:
IDisposible makes it possible to create a scope with using and force the program to call Finish when the scope is finished.
```
using(tracer.StartSpan())
{
   //Do something with the span.
} // Finish on the span is called automatically, even if an exception is thrown.
```
It is mainly used for managed resources, like files that needs to be closed. I think it is called `AutoCloseable` in Java.